.eslintrc

```diff
@@ -78,7 +79,12 @@
     ],
     "simple-import-sort/imports": "error",
     "simple-import-sort/exports": "error",
-    "func-style": ["error", "declaration"]
+    "func-style": ["error", "declaration"],
+    "jest/no-disabled-tests": "warn",
+    "jest/no-focused-tests": "error",
+    "jest/no-identical-title": "error",
+    "jest/prefer-to-have-length": "warn",
+    "jest/valid-expect": "error"
```

Comment:
What about [extending the recommenced](https://github.com/jest-community/eslint-plugin-jest/blob/main/README.md#recommended) Jest config instead of toggling rules separately?